docs/UserGuide.md

```diff
@@ -426,7 +445,15 @@ Format: `delete_session n/STUDENT_NAME i/SESSION_INDEX`
 </div>
 
 Example:
-* `delete_session n/John Doe i/1` deletes John Doe's **first** session based on the session list in the Tuition page
+1. You would like to delete an existing session for **John Doe** on the date **2021-01-01** at **18:00**.
```

Comment:
Actually now that I tink about it. Maybe we want to have an example 2 where it deletes an entire recurring session. If not adding tip below that we can use this command to delete the entire session might be good also. What do you think.

Reply:
Added a new example to `delete_session`